docs/how-to/add-custom-domain-for-an-app.md

```diff
@@ -0,0 +1,55 @@
+---
+# slug: / # if you wanna drop straight into docs uncomment along with config change
+sidebar_position: 2
+---
+
+# Adding a custom domain to an app
+
+Cyclic apps are given random subdomain names at first deployment. The general structure of a name is -
+
+`some-random-words.cyclic.app`
+
+You can customize this domain in two ways:
+
+## Change the subdomain
+
+To change the subdomain to something like `myproject.cyclic.app` , navigate to `Environments` and use the form in the `Custom Subdomain` panel. You will be able to select a any custom domain that hasn't already been used.
+
+## Add a domain you own to an app
+
+You can attach a domain you own to a Cyclic app. This requires you to verify that you own you domain in order for Cyclic to request the issue of an SSL certificate. The certificates are signed by AWS and are auto-renewed as long as appropriate DNS records exist.
```

Comment:
Perhaps add the two parts:

`You can attach a domain you own to a Cyclic app. This has two DNS records. The first verifies that Cyclic may request an SSL certificate for that domain. The certificates are signed by AWS and are auto-renewed as long as appropriate DNS records exist. The second routes requests to that domain to your Cyclic app.`